test/testcafe/spec/SessionStorage_spec.js

```diff
@@ -261,3 +263,62 @@ test.requestHooks(credentialSSONotExistLogger, credentialSSONotExistMock)('shall
   )).eql(1);
   await t.expect(getStateHandleFromSessionStorage()).eql(null);
 });
+
+test.requestHooks(identifyChallengeMock)('shall back to sign-in and authenticate succesfully', async t => {
+  const identityPage = new IdentityPageObject(t);
+  const challengeEmailPageObject = new ChallengeEmailPageObject(t);
+  let pageTitle;
+
+  // Add mocks for interaction code flow
+  const challengeSuccessMock = RequestMock()
+    .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+    .respond(xhrSuccessWithInteractionCode)
+    .onRequestTo('http://localhost:3000/oauth2/default/v1/token')
+    .respond(xhrSuccessTokens);
+  await t.addRequestHooks(challengeSuccessMock);
+
+  // Setup widget with interaction code flow
+  const optionsForInteractionCodeFlow = {
```

Comment:
It looks like this may have been copied from `EnglishLeaks_spec`. This config appears to be incorrect / misleading.

- `codeVerifier` is not a widget option. (It may exist in transaction meta)
- `codeChallenge` option will put the widget into a different mode (remediation, meant for server-side web apps). In this mode the widget will not retrieve tokens. Although the token endpoints are mocked for success, I believe this part of the flow is not being tested.
- `codeChallengeMethod` default value is `S256`. Recommend not setting this option in the test

It would be great if you could clean up the config here and in `EnglishLeaks`:
- remove `codeVerifier`, `codeChallenge` and `codeChallengeMethod` options. This should not break the tests. If, for some reason, removing these options does cause any change, we should test both cases. (with/without codeChallenge)

Reply:
Done. It has not broken tests